### PR TITLE
[Task] Update strings for thanks pop-up

### PR DESCRIPTION
**Pull request by zag2me**
Make the reputation pop-up say thanks instead

### CODE REVIEW

**Comment by zag2me** [comment]
@da-anda was easier to re-create it in the end ;)

**Comment by da-anda** [comment]
you coulde have force-pushed your new local branch to your existing github branch in order to update the PR ;)

**Comment by zag2me** [comment]
Think its best I stay away from any forced pushing haha ;)

File: xbmc/languages/english/xbmc.lang.php [hunk 1]
```diff
@@ -18,7 +18,7 @@
 $l['postbit_pm'] = 'pm';
 $l['postbit_qdelete_title'] = 'Delete this post';
 $l['postbit_qdelete'] = 'X';
-$l['postbit_reputation_title'] = 'Add a reputation rating for this user';
+$l['postbit_reputation_title'] = 'Add thanks for this user';
```
Comment (da-anda) [inline xbmc/languages/english/xbmc.lang.php]: Doesn't this read odd? I'm no native speaker, but "Say thanks to this user" might be better? Same with other changed labels. But if it's fine for you as native speaker I'm ofc fine with it. (sorry I didn't notice before)

Comment (zag2me) [inline xbmc/languages/english/xbmc.lang.php]: Yeh you are partially correct for the description, I'll amend. "Add Thanks" works when clicking the button though as you are preforming a process.

Gives me a chance to try the Amend thing now.

**Comment by zag2me** [comment]
Managed it! :beer:

**Comment by zag2me** [comment]
@da-anda any ideas where this string is based in the forum code? I had a quick look at the custom theme but couldn't work it out.

image1

**Comment by da-anda** [comment]
@zag2me I debugged this a bit and this part is used from the default theme, so it's not altered by the xbmc_theme file and thus you don't see it there (the xbmc_theme.xml only contains altered stuff). This particular line is the postbit_reputation template section. It should be using $l['reputation'] as label.

**Pull request closed without merging**